artemis-server/src/main/java/org/apache/activemq/artemis/core/protocol/ProtocolHandler.java

```diff
@@ -152,7 +152,13 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
             HttpHeaders headers = request.headers();
             String upgrade = headers.get("upgrade");
             if (upgrade != null && upgrade.equalsIgnoreCase("websocket")) {
-               ctx.pipeline().addLast("websocket-handler", new WebSocketServerHandler(websocketSubprotocolIds, ConfigurationHelper.getIntProperty(TransportConstants.STOMP_MAX_FRAME_PAYLOAD_LENGTH, TransportConstants.DEFAULT_STOMP_MAX_FRAME_PAYLOAD_LENGTH, nettyAcceptor.getConfiguration())));
+               int stompMaxFramePayloadLength = ConfigurationHelper.getIntProperty(TransportConstants.STOMP_MAX_FRAME_PAYLOAD_LENGTH, -1, nettyAcceptor.getConfiguration());
+               if (stompMaxFramePayloadLength != -1) {
+                  ActiveMQServerLogger.LOGGER.deprecatedConfigurationOption(TransportConstants.STOMP_MAX_FRAME_PAYLOAD_LENGTH);
```

Comment:
I have a small question about this, the method "deprecatedConfigurationOption" was not being used in any other place, I used it instead of creating a specify log line but I am not sure what is the "default"

Reply:
@ldebello what you mean by default here? I lost context .

Reply:
It's fine regarding the unused method being used now.  but I'm not sure what default you are referring to.

Reply:
sorry I meant which is the standard for logging things in artemis, anyway if using that method is ok, great